### ISSUE TITLE
Remove references to Angular 2 in favor of Angular 

### ISSUE THREAD

**Issue by beeman** [mean-expert-official/loopback-sdk-builder#403]
#### What type of issue are you creating?
- [x] Enhancement

#### What version of this module are you using?
- [x] GitHub Master

#### Please add a description for your issue:

Throughout the codebase there are references to Angular 2. Since we all know that `It's just Angular` I think we should rename them.

I'm happy to send in a PR, I'd like to use this issue to make sure we're using the right references.

My suggestion is to rename the references as follows:

```
Angular 2       Angular
NG2Finder       AngularFinder
Ng2webPage      AngularwebPage
angular2        angular
ng2finder       angular-finder
ng2native       angular-native
ng2native 2     angular-native
ng2universal    angular-universal
ng2web          angular-web
```

CC @jonathan-casarrubias @kattsushi

**Comment by lygstate** [comment]
AngularWebPage

Option 1

**Comment by kattsushi** [comment]
Option 2 works for me (y)

**Comment by jonathan-casarrubias** [comment]
@beeman changing commands will immediately crash a lot of projects, from building the SDK to CI crashes.

We can not simply decide to change one for the other, that would be irresponsible to other dev users from our side.

If we want to get rid of the Angular 2 references, we would need a transition for that and support both commands for some time, add documentation for that transition and give them a grace period so they can do the changes at their time without breaking anything from our side.

For me this is not a priority, so if you want to owner this that would be great, just consider is not just changing ones for the others and also that needs to be visible and well documented.

**Comment by kattsushi** [comment]
Yes, I think you should have a next version of the transition supporting previous commands and the next customizable release, of course in the documentation you have a good point!

**Comment by beeman** [comment]
@jonathan-casarrubias thanks for explaining, that's clear. 

Am I correct that this are the only 'public-facing' options: `angular2`, `ng2web`, `ng2native`, `ng2universal` ?

Also, what would be your preference, option 1 or 2?

**Comment by jonathan-casarrubias** [comment]
@beeman yes these are the only public commands that might break the projects.

I believe **option 1 is going to be best**, if we add react support, it would be odd to completely write react, but a shortener for angular.

Also, choosing the actual name is less probable that we will need to change that again in a future.

Regards
Jon
